### PR TITLE
Fix Avatar apps warnings. 

### PR DESCRIPTION
**Pull request by vladest**
Added notification for HMD.active property to make sure value will be changed

Test plan:
- run interface
- open Avatar app
- check logs for warnings, related to avatar app

https://highfidelity.fogbugz.com/f/cases/20357/AvatarApp-causes-QML-warnings-in-logs

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15828/HighFidelity-Beta-PR14556-c05e3e7.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15828/HighFidelity-Beta-PR14556-c05e3e7.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15828/HighFidelity-Beta-Interface-PR14556-c05e3e7.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15828/HighFidelity-Beta-PR14556-c05e3e7.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15828/HighFidelity-Beta-Interface-PR14556-c05e3e7.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/15976/HighFidelity-Beta-PR14556-63aa274.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15976/HighFidelity-Beta-PR14556-63aa274.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/15976/HighFidelity-Beta-Interface-PR14556-63aa274.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15976/HighFidelity-Beta-PR14556-63aa274.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/15976/HighFidelity-Beta-Interface-PR14556-63aa274.exe

File: libraries/display-plugins/src/display-plugins/AbstractHMDScriptingInterface.h [hunk 1]
```diff
@@ -15,7 +15,7 @@
 // These properties have JSDoc documentation in HMDScriptingInterface.h.
 class AbstractHMDScriptingInterface : public QObject {
     Q_OBJECT
-    Q_PROPERTY(bool active READ isHMDMode)
+    Q_PROPERTY(bool active READ isHMDMode NOTIFY activeChanged)
```
Comment (sethalves) [inline libraries/display-plugins/src/display-plugins/AbstractHMDScriptingInterface.h]: how is activeChanged different from mountedChanged?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16035/HighFidelity-Beta-PR14556-84fcf5b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16035/HighFidelity-Beta-Interface-PR14556-84fcf5b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16035/HighFidelity-Beta-PR14556-84fcf5b.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16035/HighFidelity-Beta-Interface-PR14556-84fcf5b.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16124/HighFidelity-Beta-PR14556-8d9cff3.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16124/HighFidelity-Beta-PR14556-8d9cff3.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16124/HighFidelity-Beta-Interface-PR14556-8d9cff3.dmg

**Comment by JeffClinton** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/16364/HighFidelity-Beta-PR14556-8d9cff3.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16364/HighFidelity-Beta-PR14556-8d9cff3.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/16364/HighFidelity-Beta-Interface-PR14556-8d9cff3.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16364/HighFidelity-Beta-PR14556-8d9cff3.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/16364/HighFidelity-Beta-Interface-PR14556-8d9cff3.exe

**Comment by JeffClinton** [comment]
Currently testing

**Pull request merged**